tests/test.c

```diff
@@ -5516,7 +5516,9 @@ void test_fail0 (const char *file, int line, const char *function,
         char timestr[32];
         time_t tnow = time(NULL);
 
-#ifdef _MSC_VER
+#ifdef __MINGW32__
+        strftime(timestr, sizeof(timestr), "%a %b %d %H:%M:%S %Y\n", localtime(&tnow));
```

Comment:
nit: no need for the newline (but it is ok), the removal below is optional

Reply:
OK. Side note: IIRC the `strftime` was only necessary on older versions of MinGW. They implemented `ctime_s` correctly at some point. I will leave it there for wider compatibility.